README.md

```diff
@@ -281,3 +281,38 @@ Run the command below to generate a list of infrastructure values. These values
 ```
 
 This command will output a set of key-value pairs, extracted from the infrastructure setup, that can be used as inputs in the domino.yaml configuration file.
+
+
+## Domino Backups
+To make our backups safe we could use cross account backups (Accounts under same AWS Organization), in case you want to apply this you can use the following [module](https://github.com/dominodatalab/terraform-aws-domino-backup) to enabled this and the following steps:
```

Comment:
Are they required to be under the same org?
```suggestion
(Accounts under same AWS Organization)
```

Reply:
yes, the accounts should be under the same aws organization (https://docs.aws.amazon.com/aws-backup/latest/devguide/create-cross-account-backup.html)